styles/module.css

```diff
@@ -0,0 +1,51 @@
+/* Class and ID styling following SMACSS CSS styling rules*/
+
+
+#social-bar {
+  display: inline-block;
+  padding: 15px 0;
+}
+
+
+.project h1 {
+  font-family: 'Bungee Shade', sans-serif;
+  font-size: 25px;
+}
+
+.project p {
+  padding-top: 40px;
+  font-family: 'Bungee', sans-serif;
+}
+.course p {
+  font-family:
```

Comment:
watch out for leaving partially done CSS, it could cause issues when your page is rendering and is difficult to find